### ISSUE TITLE
Command mute list (/msg, /message, ...)

### ISSUE THREAD

**Issue by xXSchrandXx** [lucavinci/bungeeban#9]
Hi, I want to ask if you can add a blocked-commands section to your config.yml and the block Statement in the ChatEvent of bungeecord.

It also would be cool if you can add a Bukkit Version wich does send a pluginmessage to your bungeeplugin. (useful for NCP, AAC, antihackplugins)
(For Setting both in 1 plugin just change the plugin.yml to a bungee.yml and a normal plugin.yml for bukkti server)

**Comment by lucavinci** [comment]
Yes I will do that. Keep in mind I am quite busy with school because it is my final year. But I will definately work on that when I have the time! :)
